python/GafferTest/DependencyNodeTest.py

```diff
@@ -691,7 +692,10 @@ def affects( self, input ) :
 		self.assertEqual( len( mh.messages ), 1 )
 		self.assertEqual( mh.messages[0].level, IECore.Msg.Level.Error )
 		self.assertEqual( mh.messages[0].context, "BadAffects::affects()" )
-		self.assertEqual( mh.messages[0].message, "TypeError: No registered converter was able to extract a C++ reference to type Gaffer::Plug from this Python object of type NoneType\n" )
+		m = "TypeError: No registered converter was able to extract a C++ reference to type Gaffer::Plug from this Python object of type NoneType\n"
+		if os.name == "nt" :
+			m = "TypeError: No registered converter was able to extract a C++ reference to type class Gaffer::Plug from this Python object of type NoneType\n"
+		self.assertEqual( mh.messages[0].message, m )
```

Comment:
Where the expected message just differs a _tiny_ bit between platforms, I think I'd prefer to use `assertRegex()`. I think the regex will make it clearer that the messages are expected to be basically the same, but with leeway in a specific place. And it's one less platform-based conditional...

Reply:
Fixed in 507c5f75fc604fb056e06e73d625e10f38a4a928, and related in fcd16c9ae5946f9fffae9b453bd17759e4adff9f